scripts/getLostAndFoundRedeemableNFTTypes.cdc

```diff
@@ -0,0 +1,38 @@
+import FindLostAndFoundWrapper from "../contracts/FindLostAndFoundWrapper.cdc"
+import NonFungibleToken from "../contracts/standard/NonFungibleToken.cdc"
+import FIND from "../contracts/FIND.cdc"
+
+pub fun main(user: String, specificType: String?) : Report {
```

Comment:
Have you talked to the frontend team about their needs for scripts here?  How do you get all the items a user has using these scripts? or al the types they can have?